server/package.json

```diff
@@ -40,7 +38,6 @@
     "@qdrant/js-client-rest": "^1.9.0",
     "@xenova/transformers": "^2.14.0",
     "@zilliz/milvus2-sdk-node": "^2.3.5",
-    "archiver": "^5.3.1",
```

Comment:
We dont do zip of db anymore which is probably where this came from

Reply:
https://github.com/Mintplex-Labs/anything-llm/commit/6d651db6d3b585e398ce371d00a21190e816f799